crates/trigger/tests/triggers/dummy.wasm

```diff
@@ -0,0 +1 @@
+Ha ha!  I'm not really a Wasm file at all
```

Comment:
:scream:

Reply:
Instead of checking in this file, should we just create a tmpfile in line during each test case?

Reply:
We could... it feels like extra complexity that a reader of the test would need to decode though...  I can make the message in the file more explanatory if that's the concern?  (The issue is that validation fails if the file doesn't exist, so I can make it say that.)

Reply:
That sounds good to me. I get what you're saying. It makes sense to have a checked in test app with real files or to completely write the files to a tempdir as we do [in `test_app()`](https://github.com/fermyon/spin/blob/76ed50dc2b6f9853fe82db1d5827dbc49abc6679/crates/trigger/src/locked.rs#L236). A hybrid approach may be messy.